### PR TITLE
Shizuru Font: Version 1.000 added

### PR DESCRIPTION
**Pull request by aaronbell**
690f52e: [gftools-packager] Shizuru Font: Version 1.000 added

* Shizuru Font Version 1.000 taken from the upstream repo https://github.com/shibuyafont/shizuru-font.git at commit https://github.com/shibuyafont/shizuru-font/commit/a03af0c110a8ac65c79fdb1b67df89cd5cfffda8.

 4c8cf76: [gftools-packager] ofl/shizurufont remove METADATA "source".  google/fonts#2587

### CODE REVIEW

File: ofl/shizurufont/DESCRIPTION.en_us.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<p>
+    About Lineup of Shibuya font
```
Comment (RosaWagner) [inline ofl/shizurufont/DESCRIPTION.en_us.html:2]: This line to remove

**Comment by RosaWagner** [comment]
"font" to remove from the name.

File: ofl/shizurufont/DESCRIPTION.en_us.html (hunk 1 above)
Comment (aaronbell) [reply ofl/shizurufont/DESCRIPTION.en_us.html]: Done

**Comment by aaronbell** [comment]
Will update PR once new descriptions are ready.

**Comment by mintoming** [comment]
New description here:

This is a font based on the handwritten sketch by handicapped artists, refined & created by design major students. The unique lines create its exceptional atmosphere. This is one of the work of Shibuya Font project, a collaboration of design major students and handicapped artist living in Shibuya city, officially approved by Shibuya city in Tokyo. 
http://www.shibuyafont.jp

**Comment by aaronbell** [comment]
@m4rc1e Ready for review

**Comment by m4rc1e** [comment]
please rename the dir to "shizuru"

**Review by m4rc1e — CHANGES_REQUESTED** [review]
Please rename the dir to shizuru

**Comment by aaronbell** [comment]
@m4rc1e Done!

**Comment by davelab6** [comment]
@m4rc1e can this be merged?

**Pull request merged**